latch_cli/docker_utils/__init__.py

```diff
@@ -0,0 +1,212 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+from pathlib import Path
+from textwrap import dedent
+from typing import List
+
+import yaml
+
+from latch_cli.constants import latch_constants
+from latch_cli.types import LatchWorkflowConfig
+
+
+class DockerCmdBlockOrder(Enum):
+    FIRST = auto()
```

Comment:
```suggestion
    first = auto()
```
no screaming if we can